README.md

```diff
@@ -284,6 +284,11 @@ lein modules itest --skip-meta :in-memory-db
 If you want to run tests against Oracle, bring up the Oracle VM and execute
 the following to create the users and run the migrations:
 
+```sh
+lein migrate
+```
+
+Deprecated commmand
```

Comment:
https://wiki.earthdata.nasa.gov/display/CMR/Oracle+DB+setup+for+local+CMR+development is the wiki page that includes instructions on how to setup the oracle database for development.  This wiki page states that we should use cmr setup db.  Why is it being deprecated?  - if the way to setup the database is changing, please update the documentation so that developers down the road can do it easily.